kommunicate/src/main/java/com/applozic/mobicomkit/api/HttpRequestUtils.java

```diff
@@ -405,6 +405,55 @@ public String getResponse(String urlString, String contentType, String accept, b
         return null;
     }
 
+    public String makePatchRequest(String stringUrl, String data) throws Exception {
```

Comment:
is it same code as other GET and POST call? can you move some common code to a separate common function, or may be pass the request method as a parameter if rest of the code is same

Reply:
Ok

Reply:
Did the code cleanup. Added single point entry for request.